source/tutorials/importing.rst

```diff
@@ -121,6 +124,89 @@ Importing data
      --input-path emp-paired-end-sequences \
      --output-path emp-paired-end-sequences.qza
 
+.. _`multiplexed barcode in seq import`:
+
+Multiplexed single-end FASTQ with barcode in sequence
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Format description
+******************
+
+Users with multiplexed single-ended barcode in sequence reads should have:
+
+1. one ``fastq.gz`` file, containing records from multiple samples,
+2. one :doc:`metadata <metadata>` file with a column of per-sample barcodes for use in FASTQ demultiplexing
+
+Obtaining example data
+```````````````````````
+
+.. command-block::
+
+   mkdir muxed-se-barcode-in-seq
+
+.. download::
+   :url: https://data.qiime2.org/2020.5/tutorials/importing/muxed-se-barcode-in-seq.fastq.gz
+   :saveas: muxed-se-barcode-in-seq/sequences.fastq.gz
+
+Importing data
+**************
+
+.. command-block::
+
+   qiime tools import \
+     --type MultiplexedSingleEndBarcodeInSequence \
+     --input-path sequences.fastq.gz \
+     --output-path multiplexed-seqs.qza
+
+In this format, sequence data is still multiplexed (i.e. you have only one ``fastq.gz`` file containing raw data for all of your samples).
+
+Because you are importing a single file of sequences, your filenames may be whatever you prefer.
+
+Each barcode need be represented only once in the metadata mapping file. Barcodes are not required to be in any specific order.
```

Comment:
This is a little confusing upon first read, maybe:

> There should be exactly one unique barcode for each sample you wish to demultiplex. Barcodes are not required to be in any specific order.

Also, we generally don't use the term "mapping" in QIIME 2, that is something we have tried to leave behind in the QIIME 1 world.

Reply:
I like your language for now. Do you know whether there is an open issue for combinatorial dual indexing, where I can make a note to make this inclusive of unique barcode pairs?